lib/block_view_lockups.go

```diff
@@ -0,0 +1,1784 @@
+package lib
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/dgraph-io/badger/v3"
+	"github.com/golang/glog"
+	"github.com/holiman/uint256"
+	"github.com/pkg/errors"
+	"math"
+	"math/big"
+	"sort"
+)
+
+//
+// TYPES: LockedBalanceEntry
+//
+
+type LockedBalanceEntry struct {
+	HODLerPKID                  *PKID
+	ProfilePKID                 *PKID
+	ExpirationTimestampNanoSecs int64
+	BalanceBaseUnits            uint256.Int
+	isDeleted                   bool
+}
+
+type LockedBalanceEntryKey struct {
+	HODLerPKID                      PKID
+	ProfilePKID                     PKID
+	ExpirationTimestampUnixNanoSecs int64
+}
+
+func (lockedBalanceEntry *LockedBalanceEntry) Copy() *LockedBalanceEntry {
```

Comment:
Do you need this? If not, delete. If yes, make sure you use it rather than doing *lockedBalanceEntry, eg in the _delete function.

Reply:
Also wouldn't Decode(Encode())  be a better way of copying? Maybe ther are other places where we could do this too.

Reply:
I have it added as it's a pattern we put in other places (e.g. BalanceEntry). I doesn't seem to be critical or apart of any interface, but I also don't think it's harmful to keep.